packages/experiments/src/components/Microfeedback/Microfeedback.types.ts

```diff
@@ -0,0 +1,39 @@
+import { IStyleFunctionOrObject } from '../../Utilities';
+import { IStyle, ITheme } from '../../Styling';
+
+// Optional interface to use for componentRef. This should be limited in scope with the most common scenario being for focusing elements.
+export interface IMicrofeedback {}
+
+export interface IMicrofeedbackSlots {}
```

Comment:
I think both `IMicrofeedback` and `IMicrofeedbackSlots` can be removed.

Reply:
Fixed